tests/cover/test_direct_strategies.py

```diff
@@ -225,6 +227,29 @@ def test_produces_valid_examples_from_args(fn, args):
     fn(*args).example()
 
 
+def test_build_class_with_target_kwarg():
+    NamedTupleWithTargetField = collections.namedtuple('Something', ['target'])
+    ds.builds(NamedTupleWithTargetField, target=ds.integers())
+
+
+@checks_deprecated_behaviour
+def test_builds_can_specify_target_with_target_kwarg():
+    ds.builds(x=ds.integers(), target=lambda x: x).example()
+
+
+def test_builds_raises_with_no_target():
+    with pytest.raises(InvalidArgument):
+        ds.builds().example()
+
+
+@pytest.mark.parametrize('non_callable', [1, 'abc', ds.integers()])
```

Comment:
I guess this could be a hypothesis-style randomized test? But writing a strategy for "non-callable" didn't really seem worth it.

Reply:
Parametrize is fine here - Hypothesis is generally better at "works for all valid inputs" than "fails for all invalid inputs"; the latter space is enormous and mostly trivial anyway.